cmd/sts-handlers.go

```diff
@@ -374,6 +374,11 @@ func (sts *stsAPIHandlers) AssumeRoleWithSSO(w http.ResponseWriter, r *http.Requ
 		}
 	}
 
+	if !globalIAMSys.Initialized() {
+		writeSTSErrorResponse(ctx, w, ErrSTSNotInitialized, errServerNotInitialized)
```

Comment:
should we bring a new error here saying `ErrSTSIAMNotInitialized` ? since object layer is initialized but not IAM.

Reply:
that would be better, done